rest/taskana-rest-spring-example/src/main/resources/sql/sample-data/check-history-event-exist.sql

```diff
@@ -0,0 +1,3 @@
+SET SCHEMA TASKANA;
```

Comment:
TASKANA should be variable.

Reply:
In this case, this property is unnecessary.
Removed!

Reply:
no, it should be variable, not removed. find schemaName into the taskana-core